samples/expected_output/unitive-filename-pattern/actionwords.js

```diff
@@ -0,0 +1,213 @@
+exports.Actionwords = {
+  iAmAnInterviewer: function () {
+
+  },
+  iAmOnThePrepPage: function () {
+
+  },
+  iClickButton: function (button) {
+
+  },
+  aModalAppears: function () {
+
+  },
+  itSaysMessage: function (message) {
+
+  },
+  theP1ListsTheCompetencies: function (p1) {
+
+  },
+  theListsTheCompetenciesThatInterviewerWasAssignedInOrderOfImportance: function () {
+
+  },
+  interviewerClicksButtonToCloseThePopUp: function (button) {
+
+  },
+  iArriveToPagePage: function (page) {
+
+  },
+  iSeeTheFieldFieldWithButtonButton: function (field, button) {
+
+  },
+  iClickButtonOnTheResumePdfFile: function (button) {
+
+  },
+  theWindowCloses: function () {
+
+  },
+  aProgressBarLoadsTo100: function () {
+
+  },
+  iGetAMessageMessage: function (message) {
+
+  },
+  iReturnToPage: function (page) {
+
+  },
+  iAmAUserWhoIsAddingResumes: function () {
+
+  },
+  userDragsAndDropsResumeInField: function () {
+
+  },
+  sovrenFails: function () {
+
+  },
+  userIsNotifiedThatNoDataWasDetectedInThePDF: function () {
+
+  },
+  isPromptedForApplicantFirstNameLastNameEmail: function () {
+
+  },
+  ifTheyWantToManuallyEnterTheRestOfTheInfo: function () {
+
+  },
+  sovrenCannotParseTheResume: function () {
+
+  },
+  pNGCreationFails: function () {
+
+  },
+  defaultIconOfPDFDisplays: function () {
+
+  },
+  userClicksSaveOnParsedResume: function () {
+
+  },
+  thatEmailAlreadyHasAResumeInTheSystemOnThatJob: function () {
+
+  },
+  userIsNotifiedThatThisPersonAlreadyHasAResumeForThisJobDoYouWantToUseThatOneOrTheNewOne: function () {
+
+  },
+  iClickYesReplace: function () {
+
+  },
+  theBackEndKeepsTheOriginal: function () {
+
+  },
+  iAmAJobOwnereditorAndIAmGoingToUseUnitiveResumeReviewer: function () {
+
+  },
+  aThumbnailPreviewShowsOfTheResumeInTheField: function () {
+
+  },
+  iAmUser: function () {
+
+  },
+  iHaveSavedAtLeastOneInterviewQuestionToOneInterviewerOnBuildInterviewPageOrAtLeastOneResumeReviewed: function () {
+
+  },
+  jobHas0ApplicantsApprovedForInterview: function () {
+
+  },
+  p1ButtonIsBlueAndNotWiggle: function (p1) {
+
+  },
+  imOnTheP1Page: function (p1) {
+
+  },
+  jobIsCreated: function () {
+
+  },
+  0ResumesAdded: function () {
```

Comment:
Yes, that appears to be a bug, since that is not a valid symbol name.  They should quote those strings, I guess.